plutus-tx/src/PlutusTx/Lift.hs

```diff
@@ -113,13 +118,13 @@ unsafely ma = runQuote $ do
 
 -- | Get a Plutus Core term corresponding to the given value, throwing any errors that occur as exceptions and ignoring fresh names.
 lift
-    :: (Lift.Lift uni a, Throwable uni fun, PLC.Typecheckable uni fun)
+    :: (Lift.Lift uni a, Throwable uni fun, PLC.Typecheckable uni fun, fun ~ UPLC.DefaultFun)
```

Comment:
I'm not wild about this only working on `DefaultFun`. I think we probably do need to expose the version here. Kind of makes me want the version that puts it in the types again :/